src/PlatformSockets.c

```diff
@@ -6,6 +10,16 @@
 
 void addrToUrlSafeString(struct sockaddr_storage* addr, char* string)
 {
+    char addrstr[48];
+#ifdef __vita__
+    struct sockaddr_in* sin = (struct sockaddr_in*)addr;
+    sceNetInetNtop(addr->ss_family, &sin->sin_addr, addrstr, sizeof(addrstr));
```

Comment:
Can you just define inet_ntop to sceNetInetNtop in PlatformSockets.h like you do with other functions?